Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN git clone ${PETSC_URL} /petsc
 WORKDIR /petsc
```

Comment:
I think we will have to get PETSC_VERSION from petsc or use the hash for tagging, if we are always going to pull the latest release.